### ISSUE TITLE
Suggestion Latitude and Longitude

### ISSUE THREAD

**Issue by CyberTamilan** [masasron/g-trends#2]
Hi, it is possible to search latest trends from particular latitude and longitude :)

**Comment by masasron** [comment]
Hi @CyberTamilan, not sure about latitude and longitude and Google Trends does seem to have trends per country, or I'm missing something?

**Comment by CyberTamilan** [comment]
@masasron India Region 'TamilNadu' IN-TN, TN means TamilNadu and please collect the list of Names similar to this https://trends.google.com/trends/explore?date=now%201-H&geo=IN-TN&q=Tamil%20Movies. Also please help me to get Rising Queries based on Name?  Refer below screenshot https://ibb.co/FWvGr55

**Comment by masasron** [comment]
@CyberTamilan I got it, so as I said it's not latitude and longitude coordinates it's country/region ID
I'm working on a major refactoring for this package (moving into TS) I'll try to include this as well

Thanks!
Ron

**Comment by CyberTamilan** [comment]
@masasron can you add 'Rising Queries' on Explorer module?

**Comment by masasron** [comment]
Yes, I think that would be cool, just a heads up, it's going to be a while until I get to it.
